### ISSUE TITLE
Screen is cropped on top when set to 1.0 vertical stretch.

### ISSUE THREAD

**Issue by Catmansys** [mgba-emu/mgba#3375]
![IMG_6447](https://github.com/user-attachments/assets/08bc2858-bbdb-438c-b379-a55077343a03)
![IMG_6448](https://github.com/user-attachments/assets/99098af4-0be3-4e45-9554-df5564b57827)

**Comment by Catmansys** [comment]
Setting it to 0.9 then shows black bars on top and bottom. Needs a more decimal point increment. Like 0.9.7

**Comment by endrift** [comment]
You should first check your TV's overscan settings. That's the common cause for this.

I would need to add proper decimal input to implement that field properly (right now it's just a hard-coded set of options), which I should do but will take time.

**Comment by Catmansys** [comment]
> You should first check your TV's overscan settings. That's the common cause for this.
> 
> I would need to add proper decimal input to implement that field properly (right now it's just a hard-coded set of options), which I should do but will take time.

Yeah 1st thing I did since setting up Wii was check over scan setting on TV. But due to signal being 480p SD TV won’t let me do anything with the settings, it’s blacked out. Can only set over scan on HD signals. Which is why I completely rely on emulators scale settings to get full screen. All of em have stretch X Y axis and also move X and Y axis. But I don’t think it’s over scan issue and more of a Wii signal issue before right side and bottom are not cropped. Left side is slightly cropped but top is very cropped. Wii being completely analog signal has screen not exactly perfect dead centre or perfect full screen.
